### PR TITLE
suggestions for process on creating a user flow

### PR DESCRIPTION
**Pull request by tamari-gray**

### CODE REVIEW

**Comment by nickmeinhold** [comment]
This looks amazing!  I just took a sneak peek and it's SO good already!  Great improvements, I look forward to when it come out of Draft status 🙂

File: .contributing/the-process.md [hunk 1]
```diff
@@ -75,16 +75,49 @@ Create: 
...
+            - After adding action steps, create "input task" in asana
+
+
+        3. Create lofi wireframe
+            - Sketch lofi wireframe on paper
+            - Create a google drawing to get input on sketch
+                - Add title: "x. *Use Case* - Lofi Sketch - *Section* (CL)"
+            - Take a photo and upload it to google drawing doc
+            - Add any helpful comments, e.g reasoning for design decisions, specific questions about design
+            - Create an "input task" in asana
```
Comment (nickmeinhold) [inline .contributing/the-process.md:99]: This is great - it's a chance to get explicit about these steps too I reckon (ie. the Asana part).  What if we said something like: 
- create a task in Asana with title "IS: Lofi Sketch < x. Use Case" and 
  - add a link to the google drawing in the task description 
  - move the task to the "Blockers & Reviews" column 
  - add a due date of tomorrow

Comment (nickmeinhold) [reply .contributing/the-process.md:99]: IS is Input Session, what do you think?

File: .contributing/the-process.md [hunk 2]
```diff
@@ -75,16 +75,49 @@ Create: 
...
+            - After 24 hr input time has ended, resolve any conversations (use real time chat if need be)
```
Comment (nickmeinhold) [inline .contributing/the-process.md:100]: What do you think if we said: 
At standdown: 
- resolve any conversations (add RTCs for the next day if need be) 
At next standup: 
- remind everyone the doc will be sealed at EOD 
At next standdown: 
- give everyone a final warning 
- resolve any conversations (use real time chat if need be) 
- declare the doc sealed

File: .contributing/the-process.md [hunk 3]
```diff
@@ -75,16 +75,49 @@ Create: 
...
+            - When all conversations are resolved, create a Hifi wireframe!
+        
 
-        -   Step 2, Part 1: Use Case → User Flow 
+        4. Create Hifi wireframe
+            - Create Hifi wireframe using Adobe XD
+            - Inside folder created in step 1, create a google drawing to get input on wireframe
```
Comment (nickmeinhold) [inline .contributing/the-process.md:106]: step 1 - same comment as above

File: .contributing/the-process.md [hunk 4]
```diff
@@ -75,16 +75,49 @@ Create: 
...
+                - Add title: "x. *Use Case* - Hifi wireframe - *Section* (CL)"
```
Comment (nickmeinhold) [inline .contributing/the-process.md:107]: Same comment as above re: ordering and using "<" over "-"

File: .contributing/the-process.md [hunk 5]
```diff
@@ -75,16 +75,49 @@ Create: 
...
+            - Export any screens and upload them to google drawing doc
```
Comment (nickmeinhold) [inline .contributing/the-process.md:108]: What do you mean by export?  I screenshot, is that exporting?

Comment (nickmeinhold) [reply .contributing/the-process.md:108]: To me it is, technically, but "export" makes me not sure I'm doing it as intended

File: .contributing/the-process.md [hunk 6]
```diff
@@ -75,16 +75,49 @@ Create: 
...
+            - Add any helpful comments, e.g reasoning for design decisions, specific questions about design
+            - Create an "input task" in asana
```
Comment (nickmeinhold) [inline .contributing/the-process.md:110]: Same as previous comment re: asana tasks 

- one additional comment - I wonder if we could have a separate definition somewhere rather than repeat the text?

File: .contributing/the-process.md [hunk 7]
```diff
@@ -75,16 +75,49 @@ Create: 
...
+            - After 24 hr input time has ended, resolve any conversations (use real time chat if need be)
```
Comment (nickmeinhold) [inline .contributing/the-process.md:111]: Same as previous comment re: 24 hour period and standdowns 

- and same sub-comment as in last comment: could we have this defined separately and refer to it rather than repeat?

File: .contributing/the-process.md [hunk 8]
```diff
@@ -75,16 +75,49 @@ Create: 
...
+            - When all conversations are resolved, Create User flow doc!
+
+        5. Create User flow doc
+            - create a google drawing file inside use case folder
```
Comment (nickmeinhold) [inline .contributing/the-process.md:115]: I think some extra specification would be good here re: the folder (and this might be good for everywhere the folder is referenced, eg. as "from step 1".  we could use: 
The "x. *Use Case* - User Flow" folder 
?

File: .contributing/the-process.md [hunk 9]
```diff
@@ -75,16 +75,49 @@ Create: 
...
+              - Add title: "x. *Use Case* - User flow - *Section* (CL)"
```
Comment (nickmeinhold) [inline .contributing/the-process.md:116]: same as previous comment re: order and use of "<"

File: .contributing/the-process.md [hunk 10]
```diff
@@ -75,16 +75,49 @@ Create: 
...
+            - Combine action steps with wireframes in file
```
Comment (nickmeinhold) [inline .contributing/the-process.md:117]: I feel like this could be fleshed out a bit but I reckon leave it for another PR if we feel like it at some point

File: .contributing/the-process.md [hunk 11]
```diff
@@ -75,16 +75,49 @@ Create: 
...
+            - Create an "input task" in asana
```
Comment (nickmeinhold) [inline .contributing/the-process.md:118]: same comment re: asana tasks

File: .contributing/the-process.md [hunk 12]
```diff
@@ -75,16 +75,49 @@ Create: 
...
+            - After 24 hr input time has ended, resolve any conversations (use real time chat if need be)
```
Comment (nickmeinhold) [inline .contributing/the-process.md:119]: same comment re: 24 hour period and standdowns

**Review by nickmeinhold — CHANGES_REQUESTED** [review]
Wooo!  Awesome work mate!!! 

I made some comments with questions and suggestions, all just meant for starting a discussion. 

Thanks for improving The Process doc!!!

File: .contributing/the-process.md [hunk 13]
```diff
@@ -75,16 +75,49 @@ Create: 
 
     -   A User Flow 
 
-        -   Step 1: Make a doc named: "x. *Use Case* < User Flow < *Section* (CL)"
+        1. Create a folder to hold all files needed to create a user flow
+            - Add title: "x. *Use Case* - User Flow files - *Section* (CL)"
+
+        2. Inside folder created in step 1, create a doc to outline action steps for the use case.
```
[the remark this answers is not in the document]
Comment (tamari-gray) [reply .contributing/the-process.md:81]: yup im happy with part a

File: .contributing/the-process.md [hunk 14]
```diff
@@ -75,16 +75,49 @@ Create: 
...
+                -   wants to...
+                    -   (goal) 
+                -   so they...
+                    -   (action steps)
+            - After adding action steps, create "input task" in asana
+
+
+        3. Create lofi wireframe
+            - Sketch lofi wireframe on paper
+            - Create a google drawing to get input on sketch
```
[the remark this answers is not in the document]
Comment (tamari-gray) [reply .contributing/the-process.md:95]: yup, gonna make that clearer! 😸

File: .contributing/the-process.md [hunk 15]
```diff
@@ -75,16 +75,49 @@ Create: 
...
+                - Add title: "x. *Use Case* - Lofi Sketch - *Section* (CL)"
```
[the remark this answers is not in the document]
Comment (tamari-gray) [reply .contributing/the-process.md:96]: idm implementing that!

[the remark this answers is not in the document]
Comment (tamari-gray) [reply .contributing/the-process.md:96]: implemented :)

File: .contributing/the-process.md [hunk 16]
```diff
@@ -75,16 +75,49 @@ Create: 
...
+            - Take a photo and upload it to google drawing doc
```
[the remark this answers is not in the document]
Comment (tamari-gray) [reply .contributing/the-process.md:97]: yeep done :)

File: .contributing/the-process.md (hunk 1 above)
Comment (tamari-gray) [reply .contributing/the-process.md:99]: @nickmeinhold love it, would it be good to have a key in the doc?
- like "IS = Input session" at the bottom of the doc

File: .contributing/the-process.md (hunk 4 above)
Comment (tamari-gray) [reply .contributing/the-process.md:107]: done :)

File: .contributing/the-process.md (hunk 5 above)
Comment (tamari-gray) [reply .contributing/the-process.md:108]: had a chat (in real life) and concluded that export is the software level export (XD) and hopefully wont be confused with screenshotting. well see when other people come on tho lol

File: .contributing/the-process.md (hunk 2 above)
Comment (tamari-gray) [reply .contributing/the-process.md:100]: love 'declare the doc sealed' 😆

File: .contributing/the-process.md (hunk 3 above)
Comment (tamari-gray) [reply .contributing/the-process.md:106]: made it clearer!

File: .contributing/the-process.md (hunk 6 above)
Comment (tamari-gray) [reply .contributing/the-process.md:110]: i was thinking that too! ill look into it!

File: .contributing/the-process.md (hunk 9 above)
Comment (tamari-gray) [reply .contributing/the-process.md:116]: implemented :)

File: .contributing/the-process.md (hunk 10 above)
Comment (tamari-gray) [reply .contributing/the-process.md:117]: sweet!

File: .contributing/the-process.md (hunk 8 above)
Comment (tamari-gray) [reply .contributing/the-process.md:115]: anything to make it clearer! im down to use that :)

File: .contributing/the-process.md (hunk 1 above)
Comment (tamari-gray) [reply .contributing/the-process.md:99]: and implemented your suggestion, thanks!

File: .contributing/the-process.md (hunk 2 above)
Comment (tamari-gray) [reply .contributing/the-process.md:100]: and i like the suggestion!

Comment (tamari-gray) [reply .contributing/the-process.md:100]: imp-lem-ent-ing :)

**Comment by tamari-gray** [comment]
Had a go at implementing those good suggestions nick! 
please point out if i missed anything or anything else to add, thank you!

File: .contributing/the-process.md (hunk 6 above)
Comment (tamari-gray) [reply .contributing/the-process.md:107]: i just made a key at the bottom of the doc for now. would love a popup or an link that scrolls to the key or something

File: .contributing/the-process.md [hunk 17]
```diff
@@ -158,4 +154,24 @@ Notes on the various tools 
...
+-   Asana Discussions are great for capturing Q&A type conversations (a bit like StackOverflow)
+
+
+Key
+====
+
+- "IS" : input session
+- "user flow folder": folder created in user flow step 1, titled: "x. *Use Case* - User Flow files - *Section* (CL)"
+- "input task": 
+  - Create a task in Asana with title "IS: Lofi Sketch < x. Use Case" and:
```
Comment (T-Davis) [inline .contributing/the-process.md:166]: Can we make this "IS: _title of Google doc_" instead

Comment (nickmeinhold) [reply .contributing/the-process.md:166]: Just to clarify - I would be happy with just adding an "eg.", and also happy with @T-Davis suggestion :-)

File: .contributing/the-process.md [hunk 18]
```diff
@@ -125,4 +154,24 @@ Notes on the various tools 
...
--   Asana Discussions are great for capturing Q&A type conversations (a bit like StackOverflow)
\ No newline at end of file
+-   Asana Discussions are great for capturing Q&A type conversations (a bit like StackOverflow)
+
+
+Key
+====
+
+- "IS" : input session
+- "user flow folder": folder created in user flow step 1, titled: "x. *Use Case* - User Flow files - *Section* (CL)"
```
Comment (nickmeinhold) [inline .contributing/the-process.md:164]: If we go with "<" over "-" we'll need to change here too

File: .contributing/the-process.md [hunk 19]
```diff
@@ -125,4 +154,24 @@ Notes on the various tools 
...
+      - Add a link to the google drawing in the task description
+      - Move the task to the "Blockers & Reviews" column
+      - Add a due date of tomorrow
+      - At standdown:
+        - resolve any conversations (add RTCs for the next day if need be)
+      - At next standup:
+        - remind everyone the doc will be sealed at end of day
+      - At next standdown:
+        - give everyone a final warning
+                resolve any conversations (use real time chat if need be)
```
Comment (nickmeinhold) [inline .contributing/the-process.md:176]: is this missing a '-' or meant to have the large indentation?

**Review by nickmeinhold — CHANGES_REQUESTED** [review]
Looks fantastic mate!  Couple of small questions and we're pretty much done I reckon! 👍

File: .contributing/the-process.md [hunk 20]
```diff
@@ -75,16 +75,45 @@ Create: 
 
     -   A User Flow 
 
-        -   Step 1: Make a doc named: "x. *Use Case* < User Flow < *Section* (CL)"
+        1. Create a folder to hold all files needed to create a user flow
+            - Add title: "x. *Use Case* - User Flow files - *Section* (CL)"
```
[the remark this answers is not in the document]
Comment (tamari-gray) [reply .contributing/the-process.md:79]: yeah it got a bit confusing 😅 im down for "<"!

File: .contributing/the-process.md (hunk 18 above)
Comment (tamari-gray) [reply .contributing/the-process.md:164]: done!

File: .contributing/the-process.md (hunk 17 above)
Comment (tamari-gray) [reply .contributing/the-process.md:166]: yess!!

File: .contributing/the-process.md (hunk 19 above)
Comment (tamari-gray) [reply .contributing/the-process.md]: missing the '-', good pickup!

File: .contributing/the-process.md (hunk 13 above)
[the remark this answers is not in the document]
Comment (nickmeinhold) [reply .contributing/the-process.md]: Sweet! (sorry didn't see this earlier)

File: .contributing/the-process.md (hunk 15 above)
[the remark this answers is not in the document]
Comment (nickmeinhold) [reply .contributing/the-process.md]: Noice!

File: .contributing/the-process.md (hunk 8 above)
Comment (nickmeinhold) [reply .contributing/the-process.md]: Legend!

File: .contributing/the-process.md (hunk 2 above)
Comment (nickmeinhold) [reply .contributing/the-process.md]: you-da-man!

File: .contributing/the-process.md (hunk 16 above)
[the remark this answers is not in the document]
Comment (nickmeinhold) [reply .contributing/the-process.md]: boom!

File: .contributing/the-process.md (hunk 3 above)
Comment (nickmeinhold) [reply .contributing/the-process.md]: Crystal! 🔮

File: .contributing/the-process.md (hunk 5 above)
Comment (nickmeinhold) [reply .contributing/the-process.md]: thanks for looping back!

File: .contributing/the-process.md (hunk 9 above)
Comment (nickmeinhold) [reply .contributing/the-process.md]: bam!

File: .contributing/the-process.md (hunk 20 above)
[the remark this answers is not in the document]
Comment (nickmeinhold) [reply .contributing/the-process.md]: Haha awesome! 👍

File: .contributing/the-process.md [hunk 21]
```diff
@@ -125,4 +154,24 @@ Notes on the various tools 
...
+
+
+Key
+====
+
+- "IS" : input session
+- "user flow folder": folder created in user flow step 1, titled: "x. *Use Case* < User Flow files < *Section* (CL)"
+- "input task": 
+  - e.g Create a task in Asana with title "IS: *title of Google doc*" and:
+      - Add a link to the google drawing in the task description
```
Comment (nickmeinhold) [inline .contributing/the-process.md:167]: Bugger, now this seems a little confusing to me - because a google doc is not a google drawing and we're using the terms interchangeably.  I don't have a good suggestion off the top of my head - could "*title of Google doc*" just be changed to *title of Google drawing*"?  Happy with other solutions, also happy if others think it's fine.

**Review by nickmeinhold — CHANGES_REQUESTED** [review]
Finishing with a super nit for consistency 🥇

File: .contributing/the-process.md (hunk 21 above)
Comment (tamari-gray) [reply .contributing/the-process.md:167]: good point! and its not always a google drawing, one of them is a google doc.

how bout "IS:  *title of google file*"?

Comment (tamari-gray) [reply .contributing/the-process.md:167]: or "IS: *title of google doc/drawing*"

Comment (nickmeinhold) [reply .contributing/the-process.md:167]: SGTM!

Comment (tamari-gray) [reply .contributing/the-process.md:167]: @nickmeinhold

Comment (nickmeinhold) [reply .contributing/the-process.md:167]: After a chat - I agree with @tamari-gray - google file sounds great!

**Review by nickmeinhold — APPROVED** [review]
So good!  Thanks for working so hard on this!  The Process is coming along beautifully!

**Review by T-Davis — CHANGES_REQUESTED** [review]
Hey got a suggestion, left it in the comments

**Review by T-Davis — APPROVED** [review]
Looks great! 👍
